<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>項目番号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_choose">
        <property name="maximum">
         <number>7</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>進めた量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_page">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="4">
       <widget class="QLabel" name="l3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QSpinBox" name="spinBox_2">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="10" column="6">
       <widget class="QSpinBox" name="spinBox_5">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>マルクス経済学</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QProgressBar" name="progressBar_1">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>経済史</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>経済数学</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ミクロ経済学</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>進捗</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QSpinBox" name="spinBox_1">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>SPI</string>
        </property>
       </widget>
      </item>
      <item row="11" column="6">
       <widget class="QSpinBox" name="spinBox_6">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>項目</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>計量経済学</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="4">
       <widget class="QLabel" name="l5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QProgressBar" name="progressBar_4">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QProgressBar" name="progressBar_5">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QProgressBar" name="progressBar_6">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QProgressBar" name="progressBar_3">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QProgressBar" name="progressBar_7">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <widget class="QSpinBox" name="spinBox_3">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="9" column="6">
       <widget class="QSpinBox" name="spinBox_4">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>TOEFL</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>項目数</string>
        </property>
       </widget>
      </item>
      <item row="12" column="6">
       <widget class="QSpinBox" name="spinBox_7">
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QProgressBar" name="progressBar_2">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QLabel" name="l4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="11" column="4">
       <widget class="QLabel" name="l6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="l1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="12" column="4">
       <widget class="QLabel" name="l7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="l2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="9" column="5">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="10" column="5">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="11" column="5">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="12" column="5">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenue">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionView"/>
   </widget>
   <addaction name="menuMenue"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
